tests/post/steps/test_volume.py

```diff
@@ -105,4 +100,38 @@ def _check_size():
         name='checking size of PersistentVolume {}'.format(name)
     )
 
-# }}}
\ No newline at end of file
+# }}}
+
+# Constants {{{
+
+
+DEFAULT_VOLUME = """
```

Comment:
This should be introduced in the next commit, because you don't need it here; this commit only introduce the helper.

Also, constants are usually defined at the top of the fileso I would move the `Constants` fold before the `Fixtures` one.